velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -245,5 +238,57 @@ TEST_F(NthValueTest, invalidFrames) {
       "k in frame bounds must be at least 1");
 }
 
+// DuckDB has errors in IGNORE NULLS logic when the start or end
+// bound is CURRENT ROW and also when both frame ends are
+// preceding or following. So the testing is limited to this
+// subset of frame clauses.
+// The limitations are fixed in the latest version of DuckDB, so
```

Comment:
Can we upgrade DuckDB?

Reply:
There is an attempt in https://github.com/facebookincubator/velox/pull/5082. But it seems like there are many issues there.